readme.md

```diff
@@ -61,6 +61,14 @@
 - [Dojo Toolkit](https://github.com/peterkokot/awesome-dojo)
 - [Inspiration](https://github.com/NoahBuscher/Inspire)
 
+## Web development
+
+- [Django](https://github.com/rosarior/awesome-django)
+- [Play1 Framework](https://github.com/PerfectCarl/awesome-play1)
+- [CakePHP](https://github.com/dereuromark/awesome-cakephp)
+- [Symfony2](https://github.com/EmanueleMinotto/awesome-symfony2)
+- [Laravel](https://github.com/tuwannu/awesome-laravel)
+- [Learn to Program](https://github.com/karlhorky/learn-to-program)
```

Comment:
@sindresorhus as to limiting myself, I want to keep the list focused and short. A long wall of links can be daunting to newcomers, who this repository is mainly aimed at.

Why I've chosen this name is simplicity. This is about the audience again. Absolute beginners will understand **Learn to Program** or **Learn to Code** better than **Learn to become a Web Developer**. Secondly, my list is focused less on what concrete skills a person will have afterwards - it's more about:
- awakening interest in programming in general
- some more advanced resources for those already in the industry

That said, you have a valid point - for people already in the industry, it's not clear what this list is offering on first glance. For this reason, I'll add a subheading to the list: `Web Development Foundation` or something.

I've moved my entry to the misc category and removed the other entries. I'll open a separate pull request for them.